lib/Conversion/TorchOnnxToTorch/OnnxLstmExpander.cpp

```diff
@@ -0,0 +1,479 @@
+#include "torch-mlir/Conversion/TorchOnnxToTorch/OnnxLstmExpander.h"
+#include "torch-mlir/Conversion/TorchOnnxToTorch/Patterns.h"
+#include "torch-mlir/Conversion/TorchOnnxToTorch/Utils.h"
+#include "torch-mlir/Dialect/Torch/IR/TorchOps.h"
+#include "torch-mlir/Dialect/Torch/Utils/Utils.h"
+
+namespace mlir::torch::onnx_c {
+
+Value createActivationByName(ConversionPatternRewriter &rewriter, Location loc,
+                             StringRef name, Value input) {
+  if (name == "Sigmoid") {
+    return rewriter.create<Torch::AtenSigmoidOp>(loc, input.getType(), input);
+  } else if (name == "Tanh") {
+    return rewriter.create<Torch::AtenTanhOp>(loc, input.getType(), input);
+  } else if (name == "Relu") {
+    return rewriter.create<Torch::AtenReluOp>(loc, input.getType(), input);
+  } else {
+    llvm_unreachable("Unsupported activation function");
+  }
+}
+
+std::pair<Value, Value>
```

Comment:
Make a simple structure for passing the information back. It makes the process of returning multiple types more straightforward as we have a more explicit naming to the returned types.